src/accordion.tsx

```diff
@@ -35,6 +35,7 @@ export const useAccordionContext = (): AccordionContextType => React.useContext(
 interface AccordionItemContentProps {
     children?: void;
     title: string;
+    titleAs?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
```

Comment:
In the case of the accordion (when a heading tag is used) [what WAI-Aria proposes](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/) is that the whole accordion button should be wrapped by the heading tag. All of the cases I've seen following this practice have only a title. I'm not sure if having multiple texts (title + subtitle) makes a difference here.

Regarding the available tags in the `titleAs` prop I also have doubts that all cases using an accordion should be represented in the heading hierarchy of the page.

For example, a [FAQ's sections should ideally use a definition list](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-faq/):

```
<dt>
  <dt><!-- Header -->
    <button>Is this an FAQ question?</button>
  </dt>
  <dd><!-- Panel -->
    Yes, it is.
  </dd>
  <dt>
    <button>What about this one?</button>
  </dt>
  <dd>
    Yap, this too.
  </dd>
  <!-- ... more questions ... -->
</dl>
```